src/Client.java

```diff
@@ -21,5 +22,141 @@ public static void main(String args[]) throws Exception{
         probably client should be interactive, and consume input with Scanner
         based on which we should make case{} for called methods
          */
+
+        /*
+        TODO:
+        1. Figure out how to create log for each manager
+        2. Figure out how to simulate several managers use the server concurrently.
+         */
+        new Client().scan();
+    }
+
+    public void scan() throws Exception {
+        Registry registry = LocateRegistry.getRegistry();
+        CenterServer stub;
+        String managerId = "";
+        boolean ifContinue = false;
+        do {
+            Scanner scanner = new Scanner(System.in);
+            System.out.println("Please input your manager ID:");
+            managerId = scanner.nextLine();
+            verifyId(managerId);
+            switch (managerId.substring(0, 3)) {
+                case "MTL": {
+                    stub = (CenterServer) registry.lookup("MTL");
+                    ifContinue = processOpt(stub);
+                    break;
+                }
+                case "LVL": {
+                    stub = (CenterServer) registry.lookup("LVL");
+                    ifContinue = processOpt(stub);
+                    break;
+                }
+                case "DDO": {
+                    stub = (CenterServer) registry.lookup("DDO");
+                    ifContinue = processOpt(stub);
+                    break;
+                }
+            }
+        } while (ifContinue);
+    }
+
+    public boolean processOpt(CenterServer stub) throws Exception {
```

Comment:
looks heavy :)
you do the "validation" of request on client side, using this master... We have it on server side also, we may go with easier way on client side also, but still, seems good idea to have it like this